FrostyCmd/Profiles/BF2042.json

```diff
@@ -0,0 +1,36 @@
+{
+  "Name": "BF2042",
+  "DisplayName": "Battlefield™ 2042",
+  "InternalName": "bf2042",
+  "Deobfuscator": "Signature",
+  "TypeInfoSignature": "",
+  "DataVersion": 20211119,
+  "FrostbiteVersion": "2021-1.1",
+  "HasStrippedTypeNames": true,
+  "Sources": [
+    {
+      "Path": "Patch",
+      "SubDirs": false
+    },
+    {
+      "Path": "Data",
+      "SubDirs": false
+    }
+  ],
+  "EbxVersion": 6,
+  "RequiresKey": true,
+  "MustAddChunks": false,
+  "EnableExecution": true,
+  "HasAntiCheat": true,
+  "EbxCompression": 9,

```

Comment:
the newer games all use oodle leviathan, check the compression method in Utils.cs in 1.0.7 for the correct ones